packages/sil-voting-app/src/components/Header.tsx

```diff
@@ -0,0 +1,34 @@
+import { Theme } from '@material-ui/core';
+import { makeStyles } from '@material-ui/styles';
+import Block from 'medulas-react-components/lib/components/Block';
+import CircleImage from 'medulas-react-components/lib/components/Image/CircleImage';
+import Typography from 'medulas-react-components/lib/components/Typography';
+import React from 'react';
+import icon from '../assets/iov-logo.svg';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  logo: {
+    flexBasis: '205px',
+    '& > div + h5': {
+      marginLeft: theme.spacing(2),
+    },
+  },
+}));
+
+const Header = (): JSX.Element => {
+  const classes = useStyles();
+
+  return (
+    <Block width="100%" minHeight="78px" display="flex" alignItems="center">
+      <Block display="flex" alignItems="center" justifyContent="center" className={classes.logo}>
```

Comment:
Why not using flexBasis directly here?

Reply:
Because Block does not have the `flexBasis` property.

Reply:
Interesting, I thought it was a lack of typing, but the actual implementation does not support it. Despite it is not the purist solution, I think is safe enough if you just declare `width="205px"` (or the size you want to apply to the left column) on the left element. Let me know how this work.

Reply:
`width="205px"` has some wrong behaviour when the screen shrinks a bit. However, `minWidth="205px"` works well. There is also no longer need for `flexGrow` in this case since it does not do anything.